### ISSUE TITLE
Division undefined

### ISSUE THREAD

**Issue by curbaczek** [buchen/portfolio#1989]
I have updated to 0.50.2 on macOS Big Sur 11.1 and now I can not open the Statement of Assets Chart. When I try to click "Chart" I get the following exception:
<img width="740" alt="image" src="https://user-images.githubusercontent.com/24271639/104854944-e961a380-5909-11eb-986a-7744a70e3703.png">

**Comment by curbaczek** [comment]
same on 0.50.3

**Comment by chirlu** [comment]
“Division undefined” means that 0/0 was attempted to be calculated. Won’t be debuggable unless you provide an example file.

**Comment by curbaczek** [comment]
I managed to solve the issue by deleting two false transactions. After deleting period blocks until the diagram was calculated successfully, I identified the two transactions with an share value of 0,00$. In this time period I have a lot of little transactions, so it was kind of tricky to find these two transactions. Unfortunately the transaction lists can not be sorted by the "per share" column, so I had to identify them near to other transactions with a value near zero. This transactions where imported a year ago and I never had problems with the visualization, so one of the last updates must lead to a different interpretation of the data. Anyhow thanks for your reply and to @buchen for his great work. I am glad that the problem could be resolved.
